### PR TITLE
Add minimal besluit template

### PR DESCRIPTION
**Pull request by lagartoverde**
Solves https://binnenland.atlassian.net/browse/GN-4237

### CODE REVIEW

**Comment by abeforgit** [comment]
Isn't this missing a pr in the frontend as well?

**Comment by lagartoverde** [comment]
No, the template service automatically picks up new templates, what are you missing?

File: standaard-templates/besluit/nieuw/minimaal-besluit.ttl [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<http://lblod.info/templates/b04fc03e-e8ff-496a-9343-1f07b4f55551> a <http://mu.semte.ch/vocabularies/ext/Template>;
+  <http://purl.org/dc/terms/title> "Minimaal besluit";
+  <http://mu.semte.ch/vocabularies/core/uuid> "b04fc03e-e8ff-496a-9343-1f07b4f55551";
+  <http://mu.semte.ch/vocabularies/ext/activeInContext> <http://data.vlaanderen.be/ns/besluit#BehandelingVanAgendapunt>;
+  <http://mu.semte.ch/vocabularies/ext/disabledInContext> <http://data.vlaanderen.be/ns/besluit#Besluit>;
+  <http://mu.semte.ch/vocabularies/ext/templateContent> """
+
+<div property="prov:generated" resource="http://data.lblod.info/id/besluiten/${generateUuid()}" typeof="besluit:Besluit ext:BesluitNieuweStijl">
+  <p>Openbare titel besluit:</p>
+  <h4 class="h4" property="eli:title" datatype="xsd:string"><span class="mark-highlight-manual">Geef titel besluit op</span></h4>
```
Comment (nvdk) [inline standaard-templates/besluit/nieuw/minimaal-besluit.ttl:10]: I guess the wrapping mark-highlight-manual span can go or is that still relevant?

**Review by nvdk — APPROVED** [review]
Looks good to me if we still need the mark-highlight-manual span

**Pull request merged**